app/models/DepartureStatus.scala

```diff
@@ -45,8 +47,9 @@ object DepartureStatus extends Enumerable.Implicits {
   case object DepartureSubmittedNegativeAcknowledgement extends DepartureStatus
 
   case object DeclarationCancellationRequestNegativeAcknowledgement extends DepartureStatus
+  private case object OtherStatus extends DepartureStatus
 
-  val values = Seq(
+  val values: Seq[DepartureStatus] = Seq(
```

Comment:
We can maybe simplify this. We probably dont need to read every status into an object, maybe just the DepartureNotification. Have a look at removing the other status' and leaving the DepartureNotification and the 'otherMessage' object